### PR TITLE
[7r3] fix sql alchemy2.0

### PR DESCRIPTION
**Pull request by chaen**
Backport https://github.com/DIRACGrid/DIRAC/pull/6739

BEGINRELEASENOTES

*DMS
CHANGE: adapt FTS3DB to sqlalchemy 2.0

*RMS
CHANGE: adapt ReqDB to sqlalchemy 2.0

ENDRELEASENOTES

### CODE REVIEW

File: src/DIRAC/WorkloadManagementSystem/Agent/VirtualMachineMonitorAgent.py [hunk 1]
```diff
@@ -159,7 +159,7 @@ def execute(self):
 
         # Get load
         avgLoad, avgRequiredSamples = self.__getLoadAvg()
-        self.log.info("Load Average is %.2f" % avgLoad)
+        self.log.info("Load Average", f"{avgLoad:.2f}")
```
Comment (fstagni) [inline src/DIRAC/WorkloadManagementSystem/Agent/VirtualMachineMonitorAgent.py]: This, and other py3 only fixes can't be in v7r3 branch.

File: src/DIRAC/RequestManagementSystem/DB/RequestDB.py [hunk 2]
```diff
@@ -44,6 +46,15 @@
     distinct,
 )
 
+try:
+    from sqlalchemy.orm import registry
+
+    sqlalchemy_mapper = registry().map_imperatively
+except ImportError:  # registry appeared in sqlalchemy 2.0
```
Comment (chrisburr) [inline src/DIRAC/RequestManagementSystem/DB/RequestDB.py:53]: Should this be:
```suggestion
except ImportError:  # registry appeared in sqlalchemy 1.4
```

File: src/DIRAC/WorkloadManagementSystem/Agent/VirtualMachineMonitorAgent.py (hunk 1 above)
Comment (chaen) [reply src/DIRAC/WorkloadManagementSystem/Agent/VirtualMachineMonitorAgent.py]: I tried and it failed, as we also run it with python 2

Comment (chaen) [reply src/DIRAC/WorkloadManagementSystem/Agent/VirtualMachineMonitorAgent.py]: for example 
```
E     File "/home/runner/work/DIRAC/DIRAC/src/DIRAC/WorkloadManagementSystem/DB/TaskQueueDB.py", line 299
E       self.log.error("Failed to insert condition", f"{field} {result['Message']}")
E                                                                                 ^
E   SyntaxError: invalid syntax
```

File: src/DIRAC/RequestManagementSystem/DB/RequestDB.py (hunk 2 above)
Comment (chaen) [reply src/DIRAC/RequestManagementSystem/DB/RequestDB.py:53]: Hum no, it's a new thing

File: src/DIRAC/Resources/Computing/SSHComputingElement.py [hunk 3]
```diff
@@ -183,7 +176,9 @@ def __ssh_call(self, command, timeout):
             except Exception as x:
                 res = (-1, "Encountered exception %s: %s" % (Exception, str(x)))
                 return S_ERROR(res)
-        else:
+        except BaseException:
```
Comment (chrisburr) [inline src/DIRAC/Resources/Computing/SSHComputingElement.py:179]: 🤢

File: src/DIRAC/RequestManagementSystem/DB/RequestDB.py (hunk 2 above)
Comment (chrisburr) [reply src/DIRAC/RequestManagementSystem/DB/RequestDB.py:53]: If it is then wouldn't it be a problem for v8r0 and integration. Plus:
```
In [1]: import sqlalchemy
   ...: from sqlalchemy.orm import registry
   ...: sqlalchemy.__version__
Out[1]: '1.4.46'
```

**Review by chrisburr — APPROVED** [review]

**Pull request merged**